index.js

```diff
@@ -235,7 +235,24 @@ module.exports.node = (scriptPath, args, options = {}) => {
 
 	const stdio = normalizeStdio.node(options);
 
-	const {nodePath = process.execPath, nodeOptions = process.execArgv} = options;
+	// Filter out `--inspect` & `--inspect-brk` from default `execArgv`.
+	const defaultExecArgv = process.execArgv.filter(arg => {
+		if (
+			arg === '--inspect' ||
```

Comment:
I think using only `arg.startsWith('--inspect')` would work here since no `process.execArgv` only targets Node CLI flags, and only `--inspect` and `--inspect-brk` start with `--inspect`. If new `--inspect-*` are added in the future, we most likely will want to exclude them as well.